### DISCUSSION TITLE
Reduce Northrend glyph costs with TBC release

### DISCUSSION THREAD

**Discussion by BeefSteaker** [chromiecraft/chromiecraft#4477] [Suggestions]
Glyphs were priced based on how long we'd get to use them.
 with once expac down and the removal of tbc glyphs from the vendor, prices of northrend glyphs should drop in price to the current TBC glyph costs

**Comment by Amrx2** [comment]
Hi Beef, we did consider this however with the earning potential in Outlands massively increasing compared to Vanilla, we will be keeping the prices the same. Appreciate the suggestion, have a good day!

Construct

**Comment by Proxig** [comment]
Hi @Amrx2 
I would like to raise another point for reducing the prices of northrend glyphs from 350G to 30G crazy i know.

With arenas coming out i thought of participating in pvp. Unfortunately i can't because it is gated by the cost of glyphs.
Shamans **MUST HAVE** glyph of stoneclaw to pvp but it costs 350g. i have 2 pve specs enhance and resto and i can't switch glyphs every time i want to PVP.

about gold and inflation, glyphs DO NOT REMOVE money from the market. I bought 2 northrend glyphs on 3 min maxed raiding characters total. it's a fraction of gold i spent on other fixed costs - flying, dual spec, gems, leg enchants and enchants in general, consumables.
High prices of northrend glyphs only prevent people from participating, which is why i would reduce the price of glyphs to about the same as the most expensive glyphs on the market.

**Comment by cachocarne** [comment]
Then, please:  Consider adjusting the list of available glyphs or offer recipes to Scribes!
https://github.com/chromiecraft/chromiecraft/discussions/5739
